tests/e2e/specs/in-game/layout.spec.js

```diff
@@ -429,3 +430,62 @@ describe.skip('Aesthetic tests', () => {
     });
   });
 });
+
+describe('Spectators Layout', () => {
+  beforeEach(() => {
+    cy.setupGameAsSpectator();
+    cy.loadGameFixture(0, {
+      p0Hand: [
+        Card.JACK_OF_CLUBS,
+        Card.JACK_OF_HEARTS,
+        Card.JACK_OF_DIAMONDS,
+        Card.JACK_OF_SPADES,
+        Card.KING_OF_HEARTS,
+      ],
+      p0Points: [],
+      p0FaceCards: [],
+      p1Hand: [Card.ACE_OF_SPADES, Card.ACE_OF_DIAMONDS, Card.ACE_OF_CLUBS],
+      p1Points: [Card.ACE_OF_HEARTS],
+      p1FaceCards: [],
+    });
+  });
+
+  it('Should display list of spectators that join', () => {
+    cy.get('[data-cy="spectate-list-button"]').should('contain', '1');
+    cy.get('[data-cy="spectate-list-button"]').click();
+    cy.get('[data-cy="spectate-list-menu"').should('contain', 'myUsername');
+    cy.get('@gameData').then((gameData) => {
+      cy.signupOpponent(playerThree);
+      cy.setOpponentToSpectate(gameData.gameId);
+      cy.get('[data-cy="spectate-list-menu"').should('contain', playerThree.username);
+      cy.signupOpponent(playerFour);
+      cy.setOpponentToSpectate(gameData.gameId);
+      cy.get('[data-cy="spectate-list-menu"').should('contain', playerFour.username);
+    });
+  });
+
+  it('Display already spectating names on join spectate', () => {
+    cy.vueRoute('/');
+    cy.signupOpponent(playerThree);
+    cy.get('@gameData').then((gameData) => {
+      cy.setOpponentToSpectate(gameData.gameId);
+      cy.signupOpponent(playerFour);
+      cy.setOpponentToSpectate(gameData.gameId);
+    });
+    cy.get('[data-cy-game-list-selector=spectate]').click();
+    cy.get(`[data-cy-spectate-game]`).click();
+    cy.get('[data-cy="spectate-list-button"]').should('contain', '3');
+    cy.get('[data-cy="spectate-list-button"]').click();
+    cy.get('[data-cy="spectate-list-menu"')
+      .should('contain', 'myUsername')
+      .should('contain', playerThree.username)
+      .should('contain', playerFour.username);
+  });
```

Comment:
Lets combine these two tests by having someone initially spectating when you join and also having a new spectator join afterwards.
```suggestion

  it('Display list of spectators and adds to list when new spectator joins', () => {
    cy.vueRoute('/');
    // Player 3 spectates player1 vs player2
    cy.signupOpponent(playerThree);
    cy.get('@gameData').then((gameData) => {
      cy.setOpponentToSpectate(gameData.gameId);
    });
    // My user begins spectating, sees player3 in spectator list
    cy.get('[data-cy-game-list-selector=spectate]').click();
    cy.get(`[data-cy-spectate-game]`).click();
    cy.get('[data-cy="spectate-list-button"]')
      .should('contain', '2')
      .click();
    cy.get('[data-cy="spectate-list-menu"')
      .should('contain', 'myUsername')
      .should('contain', playerThree.username);

    // Player 4 begins spectating
    cy.get('@gameData').then((gameData) => {
      cy.signupOpponent(playerFour);
      cy.setOpponentToSpectate(gameData.gameId);
    });
    // Player 4 now appears in spectator list
    cy.get('[data-cy="spectate-list-button"]')
      .should('contain', '3')
      .click();
    cy.get('[data-cy="spectate-list-menu"')
      .should('contain', 'myUsername')
      .should('contain', playerThree.username)
      .should('contain', playerFour.username);
  });
```